packages/strapi-design-system/src/Tag/Tag.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Text } from '../Text';
+import { Box } from '../Box';
+import { Row } from '../Row';
+
+const TagWrapper = styled(Box)`
+  border: 1px solid ${({ theme }) => theme.colors.primary200};
+  border-radius: ${({ theme }) => theme.borderRadius};
```

Comment:
I am being picky here but what do you think about creating a utility to retrieve the borderRadius that we could use in all our component? This is almost always the same code

```
const getBorderRadius = ({ theme }) => theme.borderRadius;


const Compo = styled(Box)`
  border-radius: ${getBorderRadius};
`
```

Reply:
I understand the intent of avoiding duplication. But adding more code in other places reduces visibility to me. I can make such an utility (wyt @HichamELBSI ?) but does it worth it?

Reply:
A helper for one line, not sure how useful it is. Can you explain the advantage of having this, please?

Reply:
I let you decide if you think it's worthing it